From 23f640f72022f0f2dfafabac5f6447e5fb57a386 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:26:13 -0400
Subject: [tools] perf/evlist: Fix 32-bit build error

Message-id: <1408462094-14194-205-git-send-email-jolsa@redhat.com>
Patchwork-id: 88188
O-Subject: [PATCH RHEL7.1 BZ1131394 204/325] perf evlist: Fix 32-bit build error
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 2af68ef50c6afc1632edc984e9c834545d90f597
Author: Adrian Hunter <adrian.hunter@intel.com>
Date: Fri Oct 18 15:29:07 2013 +0300

description
===========
util/evlist.c: In function 'perf_evlist__mmap':
util/evlist.c:772:2: error: format '%lu' expects argument of type 'long unsigned int', but argument 3 has type 'size_t' [-Werror=format]
cc1: all warnings being treated as errors
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/evlist.c b/tools/perf/util/evlist.c
index 30ba9fc..cd9d0f3 100644
--- a/tools/perf/util/evlist.c
+++ b/tools/perf/util/evlist.c
@@ -776,7 +776,7 @@ int perf_evlist__mmap(struct perf_evlist *evlist, unsigned int pages,
 
  evlist->overwrite = overwrite;
  evlist->mmap_len = perf_evlist__mmap_size(pages);
- pr_debug("mmap size %luB\n", evlist->mmap_len);
+ pr_debug("mmap size %zuB\n", evlist->mmap_len);
  mask = evlist->mmap_len - page_size - 1;
 
  list_for_each_entry(evsel, &evlist->entries, node) {
-- 
1.7.1